db/fixtures/miq_event_definitions.csv

```diff
@@ -193,7 +193,6 @@ containernode_compliance_failed,Container Node Compliance Failed,Default,complia
 
 containergroup_deadlineexceeded,Pod Deadline Exceeded,Default,container_operations
 containergroup_failedscheduling,Pod Failed Scheduling,Default,container_operations
-containergroup_failedsync,Pod Failed Sync,Default,container_operations
```

Comment:
@agrare I have the feeling I had to remove this as well (not sure if this has some drawbacks for existing installations out there where there may be these events on the queue already).